tests/mir-opt/reference_prop.rs

```diff
@@ -8,16 +8,31 @@
 fn opaque(_: impl Sized) {}
 
 fn reference_propagation<'a, T: Copy>(single: &'a T, mut multiple: &'a T) {
+    // CHECK-LABEL: fn reference_propagation(
+
     // Propagation through a reference.
     {
+        // CHECK: bb0: {
```

Comment:
Are these the raw underlying MIR bb numbers or also somehow renumbered?

Reply:
Raw bb numbers. I haven't bothered renumbering them.